toolkit/components/downloads/moz.build

```diff
@@ -60,7 +60,7 @@ if CONFIG['OS_ARCH'] == 'WINNT':
 
 # XXX - Until Suite builds off XULRunner we can't guarantee our implementation
 # of nsIDownloadManagerUI overrides toolkit's.
-if not CONFIG['MOZ_SUITE']:
```

Comment:
can we add MOZ_SUITE define into http://mxr.mozilla.org/mozilla-central/source/xulrunner/confvars.sh ? 
also we probably should create build tree for embedlite similar to http://mxr.mozilla.org/comm-central/source/